NGitLab.Mock/Clients/LabelClient.cs

```diff
@@ -11,49 +12,109 @@ public LabelClient(ClientContext context)
         {
         }
 
-        public Label Create(LabelCreate label)
+        public Models.Label Create(LabelCreate label)
         {
-            throw new NotImplementedException();
+            var project = FindProject(label.Id) ?? throw new InvalidOperationException($"Cannot find project with ID {label.Id}");
+            return project.Labels.Add(label.Name, label.Color, label.Description).ToClientLabel();
         }
 
-        public Label CreateGroupLabel(LabelCreate label)
+        public Models.Label CreateGroupLabel(LabelCreate label)
         {
-            throw new NotImplementedException();
+            var group = FindGroup(label.Id) ?? throw new InvalidOperationException($"Cannot find group with ID {label.Id}");
```

Comment:
You must throw specific GitLab exceptions such as NotFound. Also you need to check for permissions. (you can check how GetProject is implemented)